src/imp/auth_imp.js

```diff
@@ -7,6 +7,10 @@ export default class AuthImp {
     }
 
     getAccessToken() {
+        if (typeof this._accessToken === 'undefined' || this._accessToken === null || this._accessToken.length === 0) {
+            return 'empty';
```

Comment:
Why not return an empty string?

Reply:
empty strings are falsey in JS and it was causing an issue somewhere else with the value being interpreted incorrectly.